webapp/src/js/services/tasks-for-contact.js

```diff
@@ -51,13 +52,26 @@ angular.module('inboxServices').factory('TasksForContact',
       });
     };
 
-    var areTasksEnabled = function(docType) {
-      return RulesEngine.enabled && (docType === 'clinic' || docType === 'person');
+    var getType = function(docType) {
+      if (!RulesEngine.enabled) {
+        return Promise.resolve();
+      }
+      // must be either a person type or a leaf place type
```

Comment:
Maybe we should centralize the `leafPlaceType` somewhere.

Reply:
Good plan...